packages/runtime-dom/__tests__/directives/vModel.spec.ts

```diff
@@ -261,6 +275,16 @@ describe('vModel', () => {
     await nextTick()
     expect(data.trim).toEqual('hello, world')
 
+    trimNumber.value = '    hello, world    '
```

Comment:
I think thest tests are repeat and should add test like this.
```js
    trimAndNumber.value = ' 1 '
    triggerEvent('input', trimAndNumber)
    await nextTick()
    expect(data.trimAndNumber).toEqual(1)
```

Reply:
Thanks ! You are right 👍.